tests/test_claim_money_authority.py

```
@@ -0,0 +1,119 @@
+        facts=["Стоимость выполненных работ составляет 8 400 000 тенге."],
+        legal_basis=[],
+        requests=requests,
+        attachments=[],
+        verification_notes=[],
+    )
+
+
+def test_restores_dropped_principal_amount_only_when_same_price_is_in_source() -> None:
+    draft = _draft(
+        price="8 400 000 тенге",
+        requests=["Взыскать с Ответчика задолженность."],
```

Ruff (0.16.2)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)